src/components/StatusButton.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const StatusButton = () => (
```

Comment:
- This component should be named "Categories". Please change that.

Reply:
Hi @whiz25
That is not the Categories component. I just created a re-usable StatusCheckButton component that I'm importing in the actual Categories component. So I don't know what you want me to do.

Reply:
If you check through my files, you will find the actual Categories component